sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LogFilteringProcessor.cs

```diff
@@ -1,24 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using OpenTelemetry.Logs;
-using OpenTelemetry;
-
-namespace Azure.Monitor.OpenTelemetry.AspNetCore
-{
-    internal class LogFilteringProcessor : BatchLogRecordExportProcessor
```

Comment:
this might be worth contributing to OTel contrib repo?

Reply:
I agree, contributing this to OTel repo will help.

Reply:
or may be here
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs

Reply:
This would be an apt location with a new folder on filtering logs.